backend/apps/slack/management/commands/slack_sync_messages.py

```
@@ -0,0 +1,158 @@
+                has_more = bool(cursor)
+
+                if delay and has_more:
+                    time.sleep(delay)
+
+            self.stdout.write(
+                self.style.SUCCESS(f"Finished processing {messages_count} messages from {conversation.name}")
+            )
+
+        except SlackApiError as e:
+            self.stdout.write(
+                self.style.ERROR(f"Failed to fetch messages for {conversation.name}: {e.response['error']}")
```

Ruff (0.11.9)

Line too long (108 > 99)

(E501)